Epam.TAF.API/Controllers/PhonesController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using RestSharp;
+using Epam.TAF.API.Models.ResponseModels;
+using Epam.TAF.API.Models.RequestModels;
+
+namespace Epam.TAF.API.Controllers
+{
+    public class PhonesController : BaseController
+    {
+        public PhonesController(CustomRestClient client) : base(client, Service.Phones)
+        {
+        }
+
+        private const string ListOfAllObjectsResourse = "/objects";
+        private const string ListOfObjectsByIdResourse = "/objects?id={0}&id={1}&id={2}";
```

Comment:
this url is not really reusable, probably you can use URI Builder with query :)
https://learn.microsoft.com/en-us/dotnet/api/system.uribuilder?redirectedfrom=MSDN&view=net-7.0